packages/dev/s2-docs/src/Header.tsx

```diff
@@ -110,10 +111,40 @@ export default function Header(props: PageProps) {
             isSearchOpen={searchOpen}
             overlayId={searchMenuId} />
           <div className={style({display: 'flex', alignItems: 'center', gap: 4, justifySelf: 'end'})}>
-            <Badge variant="indigo" size="M" styles={style({marginEnd: 8})}>
-              <BetaApp />
-              <Text>Beta Preview</Text>
-            </Badge>
+            <ActionButton 
```

Comment:
Are these buttons supposed to work on the build? I noticed they strip out the commit hash when clicking them so the page doesn't actually show up